app/components/primer/alpha/overlay.pcss

```
@@ -6,7 +6,7 @@
 }
 
 /* stylelint-disable-next-line selector-pseudo-class-no-unknown */
-.Overlay[popover]:not(:open) {
+.Overlay[popover]:not(:popover-open) {
```

Should we have both selectors for a bit? So we cover browsers that don't use the new selector yet
```suggestion
.Overlay[popover]:not(:open),
.Overlay[popover]:not(:popover-open) {
```

---

We should avoid this, as `:open` is no longer a valid selector. Current browsers will throw away the whole declaration and future ones might have `:open` which will cause unintended consequences.